components/mpu60xx/idf_component.yml

```diff
@@ -0,0 +1,27 @@
+## IDF Component Manager Manifest File
+
+description: MPU60xx I2C driver
+ 
+maintainers: 
+  - "Rohan Jeet <jeetrohan92@gmail.com>"
```

Comment:
This is the most important thing we need to agree on.

If you list yourself as a maintainer, then all bugs and feature request will go your way. You should only add this, if you want to commit yourself to maintenance. It's also possible to list you as 'author' and Espressif will take care of the maintenance.

I can also see that you did not add a license to this new component. All components in this repo must be Apache-2 licensed (that is every one can use for free). If you, as author, require more strict license you will have to publish it in your own repository.

So the bottom line is: if you want to maintain, or don't want to release under Apache-2, or want to release this independently from the original mpu6050, I'd suggest releasing the component on your own.
If all of the above is false, we can review and merge in esp-bsp :)

Reply:
> @jeetrohan That's a pretty good job for your first contribution!
I learned the basics from who was considered to be one of the best in my country and I also tried to follow the guidelines in documentation and the video from VP in last year espressif DevCon. 

He (my teacher) strongly advocated opensource.
So ...
1. **Apache 2.0? : Yes**. I want my contribution to be useful to as many people as possible.
Now the only question is how do I do it? Add another file to and make another PR?(This is the only thing I can think of with my limited knowledge of git, as mentioned earlier I'm newbie on git.) 

2.  **release this independently?? :** As I mentioned in another comment its **up to you/the maintenance team**. 
I changed the _init()_  to ensure some flexibility for integrations with rest of the framework. So _**do whatever is suitable, serves the purpose**_.

3. **Author /Maintainer ?** : _Yes. It'd be great if I'm mentioned **just as Author**_ and Espressif as the maintainer takes on all the brunt of resolving the issues (:stuck_out_tongue:).  Although I would like to help resolving the problems even later, whenever possible.

 I believe my answer to Apache2.0 should clarify a lot of things in itself, but if you believe there are unresolved issues, please let me know.